### ISSUE TITLE
Mono.dll pointers/offsets do not work across computers

### ISSUE THREAD

**Issue by ih1115** [ih1115/BattleriteAimbot#4]
Tried my "static" pointers on another computer and vice versa.

Both did not work for ```Mono.dll``` pointers.

Unless you manually find your offset for the "entity list" you will have to wait for a proper solution

EDIT  : 
A possible fix for you may be https://github.com/ih1115/BattleriteAimbot/issues/7

Here is the FIX:

So recently, thanks to a youtube comment I have witnessed that if you are **not** on minimum settings or have ```Accelerate input``` enabled (under gameplay) then the entity list will not actually work for you.

So basically try setting to low and disable ```accelerate input```, restart battlerite and try again.

Reason for this may be that changing these settings effects how ```mono.dll``` appears in memory. Hence people with different settings than what I used to find the offsets will have problems.

**Comment by PerfectByte** [comment]
How are you finding your pointers? I'm also having problems finding universal static pointers.

**Comment by ih1115** [comment]
Get two players in private game (by sandboxie or otherwise).
Scan the other players x (you can use my code to get local x) multiple times until you reach around 100 results.
Pick one that has a lot of accesses, is accessed by an offset and is updated frequently.
Pointer scan depth 5.
Restart and rescan ponters to invalidate pointers etc.

Notes:
My entity list has a gap of 2C between each entity.
When Restarting and reattaching you may see that the pointers point to the value 0. This is fine since there is an entity at (0,0) sometimes. Just check if that pointer + 2c (or something) points to one of your x's.
Basically, the pointer may be pointing at some other entity instead of yours but that pointer is still correct.

**Comment by alessandrompa** [comment]
Hey buddy, what program are you using to scan X values? (sorry, kinda beginner to programming)

**Comment by ih1115** [comment]
http://www.cheatengine.org/downloads.php

**Comment by Marcondezxp** [comment]
I did it right but it is not working print: http://prntscr.com/hf1w6w http://prntscr.com/hf8rgj

**Comment by ih1115** [comment]
possible fix " working :) i changed the first box in the gameplay settings ... turn off the perfomance and easy working!﻿ "

**Comment by escitue** [comment]
tried it aswell, didn't work for me

**Comment by L30z0n** [comment]
@ih1115 I think that offsets are outdated. Could you make a video tech how to got that offsets? I'm kinda lost using cheating engine. Ill be glad. Ty and nice work here.

**Comment by ih1115** [comment]
They're working fine for a lot of people (last person that got it working that I know of was this morning).
But as mentioned in this issue, for some people, any offsets I find using ```mono.dll``` will never work (not sure why yet but it may be to do with your settings)

**Comment by L30z0n** [comment]
The aim was working good until today. Confused!

**Comment by ih1115** [comment]
That may be because of the other issue which requires restarts https://github.com/ih1115/BattleriteAimbot/issues/2

**Comment by Marcondezxp** [comment]
Can someone give me the correct settings? along with the debug?

**Comment by ih1115** [comment]
@L30z0n  did you change any settings today.
https://github.com/ih1115/BattleriteAimbot/issues/7

**Comment by Marcondezxp** [comment]
@ih1115  Can you pass me your files if your program is working?

**Comment by hppsc1215** [comment]
I have the Same Issue since today. Aimrite only spams -1.07374e+08 an does nothing ingame. Friday and the all days before all was fine. Changed nothing

**Comment by ih1115** [comment]
@hppsc1215 what branch is that on?

**Comment by Marcondezxp** [comment]
@hppsc1215  mine too, these numbers http://prntscr.com/hfgmu8

**Comment by Marcondezxp** [comment]
I am in doubt if the program is not working because of the configurations or because of my computer

**Comment by ih1115** [comment]
@Marcondezxp  I'm confused as to what branch that is because it looks like ```dev``` but title says ```master```..

Either way if restarting multiple times doesn't fix that ```local x```. That is a problem, I would assume, is specific to you.
Also just making sure but you are in game when opening this right>?

**Comment by hppsc1215** [comment]
i try with different new compiled debugs. BattleriteAimbot-master; BattleriteAimbot-dev_Jade.
With my old debugs he spams the same numbers as allways (like -0.29) but doesnt aim on enemy.

And yeah iam ingame when opening.

**Comment by Marcondezxp** [comment]
I open the game and on the home screen I open the hacker already tried to open also when the game starts but anyway nothing happens

**Comment by Marcondezxp** [comment]
I just compiled the BattleriteAimbot-master folder I do not know what is this part of: BattleriteAimbot-dev_Jade.

**Comment by Marcondezxp** [comment]
is there any step I have to select some character in the hacker for example jade?

**Comment by ih1115** [comment]
Guys don't try ```dev_Jade``` if you haven't got ```Jade``` working.
Here is ```Jade```
https://github.com/ih1115/BattleriteAimbot/tree/Jade

**Comment by hppsc1215** [comment]
now it works. ty

**Comment by Marcondezxp** [comment]
loolll worked

**Comment by Marcondezxp** [comment]
if I need to choose the character in the hack for the "master"

**Comment by ih1115** [comment]
@Marcondezxp  I don't understand your question.
For people who have ```Git``` installed ( https://git-scm.com/ ), you can just run command ```git checkout Jade``` to switch to that character.

**Comment by Marcondezxp** [comment]
what is the difference between the hack "JADE" and "DEV_JADE" ??

**Comment by ih1115** [comment]
@Marcondezxp  ```dev_Jade``` make use of more information of your player such as Cool down of abilities to select an ability.
Also does not press keys but instead directly interacts with the game.

However, it is **very** unstable.

**Comment by sd914611805** [comment]
Can you record a video on updates that can now be used normally,and NEW discordapp?
